.evergreen/resync-specs.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+# exit when any command fails
+set -e
+PYMONGO=~/Work/mongo-python-driver
+SPECS=~/Work/specifications
+
+help (){
+echo "Usage: resync_specs.sh [-bcsp] spec"
+echo "Required arguments:"
+echo " spec     determines which folder the spec tests will be copied from."
+echo "Optional flags:"
+echo " -b  is used to add a string to the blocklist for that next run. Can be used"
+echo "     any number of times on a single command to block multiple patterns."
+echo "     You can use any regex pattern (it is passed to 'grep -Ev')."
+echo " -c  is used to set a branch or commit that will be checked out in the"
+echo "     specifications repo before copying."
+echo " -s  is used to set a unique path to the specs repo for that specific"
+echo "     run."
+echo " -p  does the same thing but for the pymongo repo."
```

Comment:
This shouldn't be needed since this is script is in the pymongo repo so the script can always find the folder via a related path from this file.

Reply:
Done.